zerver/openapi/zulip.yaml

```diff
@@ -8714,6 +8714,18 @@ paths:
 
                           See [PATCH /settings](/api/update-settings) for details on
                           the meaning of this setting.
+                      send_private_typing_notifications:
+                        type: boolean
+                        description: |
+                          Present if `update_display_settings` is present in `fetch_event_types`.
+
+                          Whether the user has chosen to send typing notifications in private messages.
```

Comment:
These should have API changelog entries, `**Changes**` notes, etc.

Reply:
Also we should have "typing notifications" link to the /help/ docs for the feature.

Reply:
We don't seem to have a /help page for typing notifications. I think we only have [API docs](https://chat.zulip.org/api/set-typing-status) and [developer docs](https://zulip.readthedocs.io/en/latest/subsystems/typing-indicators.html) (We'll probably have to tweak this once we merge stream typing notifications PR)

I guess we should add one soon?

Reply:
Oh! we actually have /help [page for typing notifications](https://chat.zulip.org/help/status-and-availability#typing-notifications) but it is hidden as a subsection in [status-and-availability](https://chat.zulip.org/help/status-and-availability).

Linked "typing notifications" to those docs and rebased the PR to match to bump latest `API_FEATURE_LEVEL`.